Makefile

```diff
@@ -49,35 +44,32 @@ $(PROTOC):
 	rm $(PROTOC).zip
 	chmod 755 $(PROTOC)
 
-$(DEP):
-	mkdir -p $(TARGET)
-	$(CURL) -Lso $(DEP) $(DEP_URL)
-	chmod 755 $(DEP)
-
 Cargo.lock: Cargo.toml rs/Cargo.toml
 	$(CARGO) fetch
 
 .PHONY: rs
 rs: Cargo.lock
 	$(CARGO_TEST)
 
-.PHONY: godep
-godep: Gopkg.toml $(DEP)
-	$(DEP) ensure
+.PHONY: go-build
+go-build:
+	$(GO) build ./...
 
 .PHONY: go
-go: godep $(PROTOC)
-	$(GO) install ./vendor/github.com/golang/protobuf/protoc-gen-go
-	rm -rf go/*
+go: go-build $(PROTOC)
+	$(GO) install github.com/golang/protobuf/protoc-gen-go
+	@rm -rf go/*
```

Comment:
nit: I'd move this down below the protoc invocations -- if any of them fail, you won't have removed the prior generated files.